components/organism/nestedCheckboxes/README.md

```diff
@@ -0,0 +1,37 @@
+# OrganismNestedCheckboxes
+
+This component show a parent checkbox with children that their controls his own state.
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-organism-nested-checkboxes
+```
+
+## Usage
+
+### Basic usage
+```js
+import OrganismNestedCheckboxes from '@s-ui/react-organism-nested-checkboxes'
+
+const data = [
+  {id: 'nested-01', label: 'Nested 1', checked: true},
+  {id: 'nested-02', label: 'Nested 2', checked: true},
+  {id: 'nested-03', label: 'Nested 3', checked: true},
+  {id: 'nested-04', label: 'Nested 4', checked: false},
+  {id: 'nested-05', label: 'Nested 5', checked: false}
+]
+
+<OrganismNestedCheckboxes
+  fullCheckedStyledIcon={IconCheck}
+  halfCheckedStyledIcon={IconHalfCheck}
+  id="nested"
+  items={data}
```

Comment:
would it be to hard to use compound components? https://medium.com/@Dane_s/react-js-compound-components-a6e54b5c9992

Reply:
We are working on it in a new pull request 😉